packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -86,13 +92,30 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
   /// Scrollbar will be rectangular if [radius] is null.
   final Radius radius;
 
-  /// The smallest size the scrollbar can shrink to when the total scrollable
-  /// extent is large and the current visible viewport is small, and the
-  /// viewport is not overscrolled. Mustn't be null.
+  /// The amount of space by which to inset the scrollbar's start and end, as
+  /// well as its side to the nearest edge, in pixels.
+  ///
+  /// This is typically set to the current [MediaQueryData.padding] to avoid
+  /// partial obstructions such as display notches. If you want additonal
+  /// margins around the scrollbar, see [mainAxisMargin] or [crossAxisMargin].
```

Comment:
Oh this isn't API breaking :)

We should be very explicit about `padding` being redundant with `mainAxisMargin` and this lets you control the exact padding on each side and *margin are just doubles.